README.adoc

```diff
@@ -47,13 +52,14 @@ git clone -o upstream https://github.com/projectriff/function-sidecar src/github
 git clone -o upstream https://github.com/projectriff/function-controller src/github.com/projectriff/function-controller/
 git clone -o upstream https://github.com/projectriff/topic-controller src/github.com/projectriff/topic-controller/
 git clone -o upstream https://github.com/projectriff/http-gateway src/github.com/projectriff/http-gateway/
+git clone -o upstream https://github.com/projectriff/shell-function-invoker src/github.com/projectriff/shell-function-invoker/
+git clone -o upstream https://github.com/projectriff/riff-cli src/github.com/projectriff/riff-cli/
```

Comment:
This conflicts with the README in the riff-cli which says `go get github.com/projectriff/riff-cli`

Isn't it time to stop using "upstream" here and switch to "origin"?

Reply:
i'd prefer to be consistent with the rest of the readme just for now - and change all the go repos to `go get` (with default origin) if everyone is ok with that in a separate PR